src/views/List.jsx

```diff
@@ -1,34 +1,36 @@
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import { ListItem } from '../components';
-import { getItemData, streamListItems } from '../api';
 
-export function List({ listToken }) {
-	const [data, setData] = useState([]);
-	const [loading, setLoading] = useState(true);
+export function List({ data }) {
+	const [searchField, setSearchField] = useState('');
 
-	useEffect(() => {
-		streamListItems(listToken, (snapshot) => {
-			const nextData = getItemData(snapshot);
+	const onSearchChange = (e) => {
+		const { value } = e.target;
+		setSearchField(value.toLocaleLowerCase());
+		console.log(value);
+	};
 
-			setData(nextData);
-			setLoading(false);
-		});
-	}, [listToken]);
-
-	if (loading) {
-		return <p>Loading...</p>;
-	}
+	const filteredLists = data.filter(({ name }) =>
+		name.toLocaleLowerCase().includes(searchField),
```

Comment:
Same as above comment :slightly_smiling_face:

Reply:
Fixed this! Thanks!